app/src/main/java/org/mozilla/fenix/tabstray/inactivetabs/InactiveTabs.kt

```diff
@@ -0,0 +1,280 @@
+    onClick: () -> Unit,
+    onDeleteAllClick: () -> Unit,
+) {
+    ExpandableListHeader(
+        headerText = stringResource(R.string.inactive_tabs_title),
+        expanded = expanded,
+        expandActionContentDescription = stringResource(R.string.inactive_tabs_expand),
+        collapseActionContentDescription = stringResource(R.string.inactive_tabs_collapse_group),
+        onClick = onClick,
+    ) {
+        IconButton(
+            onClick = onDeleteAllClick,
+            modifier = Modifier.padding(horizontal = 8.dp),
+        ) {
+            Icon(
+                painter = painterResource(R.drawable.ic_delete),
+                contentDescription = stringResource(R.string.inactive_tabs_delete_all),
+                modifier = Modifier.size(20.dp),
+                tint = FirefoxTheme.colors.iconPrimary,
+            )
+        }
+    }
+}
+
+/**
+ * Inactive Tabs auto close dialog.
+ *
+ * @param onDismissClick Called when the user clicks on the auto close dialog's dismiss button.
+ * @param onEnableAutoCloseClick Called when the user clicks on the auto close dialog's enable button.
+ */
+@Composable
+private fun InactiveTabsAutoClosePrompt(
+    onDismissClick: () -> Unit,
+    onEnableAutoCloseClick: () -> Unit,
+) {
+    Card(
+        modifier = Modifier.padding(horizontal = 16.dp, vertical = 12.dp),
+        shape = ROUNDED_CORNER_SHAPE,
+        backgroundColor = FirefoxTheme.colors.layer2,
+        border = BorderStroke(
+            width = 1.dp,
+            color = FirefoxTheme.colors.borderPrimary,
+        ),
+    ) {
+        Column(
+            modifier = Modifier.padding(start = 12.dp),
+            horizontalAlignment = Alignment.End,
+        ) {
+            Row(
+                verticalAlignment = Alignment.CenterVertically,
+            ) {
+                PrimaryText(
+                    text = stringResource(R.string.tab_tray_inactive_auto_close_title),
+                    modifier = Modifier.weight(1f),
+                    fontSize = 14.sp,
+                    fontFamily = FontFamily(Font(R.font.metropolis_semibold)),
+                )
+
+                IconButton(
+                    onClick = onDismissClick
+                ) {
+                    Icon(
+                        painter = painterResource(R.drawable.mozac_ic_close),
+                        contentDescription =
+                        stringResource(R.string.tab_tray_inactive_auto_close_button_content_description),
+                        modifier = Modifier.size(20.dp),
+                        tint = FirefoxTheme.colors.iconPrimary,
+                    )
+                }
+            }
+
+            SecondaryText(
+                text = stringResource(
+                    R.string.tab_tray_inactive_auto_close_body_2,
+                    stringResource(R.string.app_name)
+                ),
+                modifier = Modifier
+                    .fillMaxWidth()
+                    .padding(end = 12.dp),
```

Comment:
I think we can avoid this by making the `Column` have a horizontal padding of 12dp. To get the `IconButton` to be at the right edge, you can specify the size(20.dp) modifier on the IconButton (not the Icon). In my testing with the accessibility scanner, it didn't complain about the tap area.